packages/@coorpacademy-app-review/src/actions/ui/quit-popin.ts

```diff
@@ -0,0 +1,25 @@
+export const OPEN_POPIN = '@@ui/OPEN_POPIN';
+export const CLOSE_POPIN = '@@ui/CLOSE_POPIN';
+
+type quitPopinPayload = {
```

Comment:
favour PascalCase for the types over camelCase
```suggestion
type QuitPopinPayload = {
```